### PR TITLE
LPD-13560

### PR DESCRIPTION
**Pull request by alessiorendina**
https://liferay.atlassian.net/browse/LPD-13560

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce-functional

**Comment by alessiorendina** [comment]
ci:test:playwright

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/rest-openapi.yaml [hunk 1]
```diff
@@ -811,6 +811,11 @@ components:
                 specificationKey:
                     example: specification-key
                     type: string
+                specificationPriority:
+                    example: 1.1
+                    format: double
+                    minimum: 0
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/rest-openapi.yaml:817]: this should be readonly

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.
Thank you!

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/rest-openapi.yaml (hunk 1 above)
Comment (alessiorendina) [reply modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/rest-openapi.yaml:817]: Changed, thank you!

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:playwright

**Comment by alessiorendina** [comment]
ci:report:2547680607

**Comment by alessiorendina** [comment]
ci:test:playwright

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:playwright

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:report:2548077772

**Comment by alessiorendina** [comment]
ci:forward:force

**Pull request closed without merging**